frontend/amp/components/Footer.tsx

```diff
@@ -6,37 +6,35 @@ import InnerContainer from './InnerContainer';
 import { footerLinks, Link } from '../../lib/footer-links';
 
 const footer = css`
-    background-color: ${palette.neutral[20]};
+    background-color: #052962;
```

Comment:
We should be adding new colours to `palette.ts` and not be including raw colour values.

Reply:
Yeah, I agree. This is a new header/footer colour - I am going to talk to @zeftilldeath about how to add this.

Reply:
Okay this is now added under `brand.blue` although happy to hear alternative naming suggestions (@zeftilldeath guardian brand.)